hw6_social.py

```diff
@@ -215,7 +212,13 @@ def getDataForRegion(data, colName):
 Returns: dict mapping strs to ints
 '''
 def getHashtagRates(data):
-    return
+    tempo={}
+    for x in data['hashtags']:
+        for j in x:
+            if (j not in tempo) and len(j)!=0:

```

Comment:
In line 218, len(j) != 0 is not required because when x is empty list it does not goes into second for loop

Reply:
issue resolved